pymc3/distributions/continuous.py

```diff
@@ -4223,6 +4223,29 @@ class Interpolated(BoundedContinuous):
     Both parameters ``x_points`` and values ``pdf_points`` are not variables, but
     plain array-like objects, so they are constant and cannot be sampled.
 
+    .. plot::
+    import matplotlib.pyplot as plt
```

Comment:
rst is very peculiar about the indexing and spacing of directives and their content. There must be a blank line between them _and_ the content must be indented.